test/testclass.cpp

```diff
@@ -2589,6 +2591,55 @@ class TestClass : public TestFixture {
         ASSERT_EQUALS("", errout_str());
     }
 
+#define checkConstructors(code) checkConstructors_(code, __FILE__, __LINE__)
```

Comment:
Shouldn't this be in `testconstructors.cpp`?

Reply:
Yes...
https://github.com/danmar/cppcheck/pull/6913